src/bandersnatch/mirror.py

```diff
@@ -42,9 +42,7 @@ class Mirror:
     now = None
 
     def __init__(
-        self,
-        master: Master,
-        workers: int = 3,
+        self, master: Master, workers: int = 3,
```

Comment:
You have to remove the trailing ',' with latest black if you want this.

Reply:
Yeah this is just an artifact from running black locally, I don't have the latest version apparently.